lib/MetaCPAN/Script/Mapping.pm

```diff
@@ -134,6 +146,11 @@ sub index_delete {
     $self->_check_index_exists( $name, EXPECTED );
     $self->are_you_sure("Index $name will be deleted !!!");
 
+    $self->_index_delete($name);
+}
+
+sub _index_delete {
```

Comment:
the "proper" names were taken by the attributes. i'll come up with some postfix to improve the methods names, i didn't like it either.